docs/portal/portal-overview.md

```diff
@@ -36,20 +37,23 @@ There are two main ways to access the pages, as shown in the following image:
 
 ![UID2 Portal Structure](images/portal-uid2-overview.png)
 
-From the left sidebar, you can access the following pages relating to configuring your account:
+Your participant name will be displayed at the top of the left sidebar. If your account has access to multiple participants, as shown in the example above, a dropdown menu will appear. You can use this menu to switch between participants in the portal. 
```

Comment:
Maybe add somewhere like on the Team Members page an explicit sentence that explains how an account does get access to a new participant outside of their default one - that an Admin for a participant needs to add someone as one of the team members for that person to get access.  Also this may be obvious, but when the admin adds them, they can add them with either Admin or Operations role to access to another participant - I'll let you make the call on that part.

Reply:
I'm a bit hesitant to add details around how to invite an existing user to a new participant because it should just be the same as adding a brand new user to a participant. I'm planning on merging as-is and we can discuss/address separately.

Reply:
@alex-yau-ttd @ashleysmithTTD looking over these mods and I'd definitely be in favor of a) defining the roles in the doc, b) explaining how someone is assigned a role. It should be covered, I think.